cif_core.dic

```diff
@@ -19286,6 +19286,593 @@ save_ATOM
+      _description_example.case
+         'Bruker'
+         'ELEMISSION'
+         'Thermo Fisher Scientific'
+
+save_
+
+save_atom_analytical_source.equipment_model
+
+    _definition.id                '_atom_analytical_source.equipment_model'
+    _definition.update            2022-11-18
+    _description.text
+
+;
+    Text describing the model of the equipment used
+    to determine the elemental composition.
+;
+    _name.category_id             atom_analytical_source
+    _name.object_id               equipment_model
+    _type.purpose                 Describe
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Text
+
+    loop_
+      _description_example.case
+         'S2 Puma'
+         'COBRA'
+         'CAP PRO XPS ICP-OES'
+
+save_
+
+save_atom_analytical_source.id
+
+    _definition.id                '_atom_analytical_source.id'
+    _definition.update            2022-11-18
+    _description.text
+
+;
+    Arbitrary label uniquely identifying the source of an elemental
+    composition value. This value is used by _atom_analytical.meas_id
+    to link individual composition values to their corresponding
+    technique of determination.
+;
+    _name.category_id             atom_analytical_source
+    _name.object_id               id
+    _type.purpose                 Key
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+
+save_
+
+save_atom_analytical_source.special_details
+
+    _definition.id                '_atom_analytical_source.special_details'
+    _definition.update            2022-11-18
+    _description.text
+
+;
+    Text describing the equipment or conditions under which the
+    data were collected that are not able to be captured using
+    _atom_analytical_source.equipment_manufacturer,
+    _atom_analytical_source.equipment_model, or
+    _atom_analytical_source.technique.
+;
+    _name.category_id             atom_analytical_source
+    _name.object_id               special_details
+    _type.purpose                 Describe
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_atom_analytical_source.technique
+
+    _definition.id                '_atom_analytical_source.technique'
+    _definition.update            2022-11-18
+    _description.text
+
+;
+    Text describing the experimental technique used to find
+    the elemental composition.
+;
+    _name.category_id             atom_analytical_source
+    _name.object_id               technique
+    _type.purpose                 Describe
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Text
+
+    loop_
+      _description_example.case
+;
+       XRF
+;
+;
+       LIBS
+;
+;
+       Inductively-coupled plasma
+       optical emission spectrometry
+;
+
```

Comment:
The value "other" could be added as one of the options with an explicit description that refers the user to the `_atom_analytical_source.special_details` data item. Similar approach is already taken with several other data items from this dictionary (see `_audit_author_role.role`, `_diffrn_measurement.specimen_attachment_type`, `_atom_sites.solution_hydrogens`, etc.).

Reply:
Kind of. The examples you give have their values set by an enumeration. In this case, it's a free-text field to enter info, as the shear number and variety of potential techniques defy enumeration. I have altered the definition of `_aas.technique` such that it should be "succinct", and that if further details are required, use `_aas.special_details`.

Reply:
I've also added some examples to `_aas.special_details`.

Reply:
Thank you for the changes. You are totally correct that this is a free-text field, I mistakenly misread it as an enumerated value.